src/Symfony/Bundle/SecurityBundle/Tests/DependencyInjection/Fixtures/xml/sodium_encoder.xml

```diff
@@ -1,16 +1,17 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<srv:container xmlns="http://symfony.com/schema/dic/security"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:srv="http://symfony.com/schema/dic/services"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services
+        https://symfony.com/schema/dic/services/services-1.0.xsd
+        http://symfony.com/schema/dic/security
+        https://symfony.com/schema/dic/security/security-1.0.xsd">
 
-<container xmlns="http://symfony.com/schema/dic/services"
-   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-   xmlns:sec="http://symfony.com/schema/dic/security"
-   xsi:schemaLocation="http://symfony.com/schema/dic/services https://symfony.com/schema/dic/services/services-1.0.xsd">
+    <srv:imports>
+        <srv:import resource="container1.xml"/>
+    </srv:imports>
```

Comment:
Can you revert the namespace changes? That would limit the diff with other branches and ease merging up patches. Thank you.

Reply:
Done!